libraries/api-extractor-model/src/mixins/ApiExtendsMixin.ts

```diff
@@ -0,0 +1,393 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.s
+
+import { ApiItem, IApiItemJson, IApiItemConstructor, IApiItemOptions, ApiItemKind } from '../items/ApiItem';
+import { ExcerptToken, ExcerptTokenKind, IExcerptTokenRange } from './Excerpt';
+import { ApiDeclaredItem } from '../items/ApiDeclaredItem';
+import { InternalError, LegacyAdapters } from '@rushstack/node-core-library';
+import { ApiModel } from '../model/ApiModel';
+import { DeserializerContext } from '../model/DeserializerContext';
+import { HeritageType } from '../model/HeritageType';
+import { IResolveDeclarationReferenceResult } from '../model/ModelReferenceResolver';
+import { ApiNameMixin } from './ApiNameMixin';
+
+/**
+ * Result object for {@link (ApiExtendsMixin:interface).findMembersWithInheritance}.
+ * @public
+ */
+export interface IFindMembersWithInheritanceResult {
+  /**
+   * The inherited members that could be found. Not guaranteed to contain all of the item's
+   * inherited members.
+   */
+  members: ApiItem[];
+
+  /**
+   * Indicates whether the members are potentially incomplete due to errors or gaps in finding
+   * inherited members. If true, the `FindMembersWithInheritanceMessageCallback` callback
+   * will be called with messages explaining the errors in more detail.
+   */
+  maybeIncompleteMembers: boolean;
```

Comment:
`missingData`? `incompleteResult`?

If I saw `maybeIncompleteMembers=true` without any docs, from the name alone I'm not sure it would be obvious that a problem occurred, or that `Members` is referring to the `FindMembersWithInheritanceResult.members` output.  (Maybe I'm being obtuse though heheh.)